bot/kodiak/evaluation.py

```diff
@@ -74,18 +74,24 @@ def get_body_content(
     strip_html_comments: bool,
     cut_body_before: str,
     cut_body_after: str,
+    cut_body_and_text: bool,
     pull_request: PullRequest,
 ) -> str:
     if body_type is BodyText.markdown:
         body = pull_request.body
+        # TODO(jbergstroem): This could probably be made faster by just cutting the string differently
```

Comment:
I don't think the performance here will be too much of an issue

Reply:
Me neither, just wanted to leave it open for you to decide.